package.json

```diff
@@ -62,10 +66,10 @@
     "flatten": "1.0.2",
     "flow-bin": "0.73.0",
     "get-port": "3.2.0",
-    "husky": "0.14.3",
+    "husky": "1.1.1",
```

Comment:
For some reason husky increased install size a lot in 1.0, let's move to https://www.npmjs.com/package/pre-commit

Reply:
I just realized pre-commit is maintained by @3rd-Eden 😄